rules-reviewed/eap7/eap6/hibernate4-xml.windup.xml

```diff
@@ -40,5 +45,44 @@
                 <matches pattern="(cfg.xml|properties)" />
             </where>
         </rule>
+        <rule id="hibernate4-00002">
+            <when>
+                <filecontent filename="{*}.hbm.xml" pattern="org.hibernate.id.{idclass}"/>
+            </when>
+            <perform>
+                <hint title="Class org.hibernate.id.{idclass} was removed/deprecated in Hibernate 5" effort="1" severity="mandatory">
+                    <message>Use `org.hibernate.id.enhanced.SequenceStyleGenerator` instead.</message>
+                    <link
+                        href="https://github.com/hibernate/hibernate-orm/blob/5.0/migration-guide.adoc#deprecations"
+                        title="Removed or deprecated types in Hibernate 5" />
+                    <link
+                        href="https://access.redhat.com/documentation/en/red-hat-jboss-enterprise-application-platform/7.0.beta/migration-guide/chapter-4-application-changes#hibernate_and_jpa_migration_changes"
+                        title="Hibernate and JPA migration changes" />
+                    <tag>hibernate</tag>
+                </hint>
+            </perform>
+            <where param="idclass">
+                <matches pattern="(TableGenerator|TableHiLoGenerator|SequenceGenerator|SequenceIdentityGenerator|SequenceHiLoGenerator)" />
+            </where>
+        </rule>
+        <rule id="hibernate4-00003">
+            <when>
+                <xmlfile 
+                 matches="//generator[@class='hilo'] | //generator[@class='sequence-identity'] | //generator[@class='seqhilo']"
+                 in="{*}.hbm.xml"/>
+            </when>
+            <perform>
+                <hint title="Identifier generator was removed in Hibernate 5" effort="1" severity="mandatory">
```

Comment:
This can't be made to show the exact identifier, like the rule above, can it? I'd prefer it to read `Generator XXX was removed/deprecated in Hibernate 5`.

If not, then at least `This generator was removed/deprecated in Hibernate 5`.